src/components/ArticleInfo.jsx

```diff
@@ -0,0 +1,27 @@
+import { useParams } from 'react-router-dom';
+
+export default function ArticleInfo({articles}){
+
+    const {article_id} = useParams()
+
+    let article;
+
+    articles.article.forEach((element)=>{
```

Comment:
You should use the endpoint for an individual article from your API, could you please submit when you have refactored this

Reply:
You may notice, this way wouldn't include the article body so the user wouldn't be able to read it